card/lib/cardio.rb

```diff
@@ -50,9 +50,8 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
+      paths.add 'tmp/set',         :with => "#{root}/tmp/set"
+      paths.add 'tmp/set_pattern', :with => "#{root}/tmp/set_pattern"
```

Comment:
Are you sure?  I guess it depends on the paths object's root, which I guess is the same thing as the deck/app root.  Well the tmp/lib one isn't needed.